ChangeLog.md

```diff
@@ -1,9 +1,21 @@
 # v0.5.0.0
 
+- Derives `Generic1` instances for all non-existentially-quantified effect datatypes.
```

Comment:
This isn’t precisely the same as “all first-order effect datatypes” because `Resumable` uses an existential and thus doesn’t admit a `Generic1` instance.